### PR TITLE
Serve Dashboard through Riposte

### PR DESCRIPTION
**Pull request by sdford**
Serve the Dashboard using Riposte instead of S3 hosting

** Note: This change won't be official until the network routing rules are updated in the Gateway/NGINX configuration.

### CODE REVIEW

File: src/main/java/com/nike/cerberus/AssetResourceFileFactory.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    /**
+     * @return The contents of the given file in bytes
+     */
+    private static byte[] getFileContents(String filePath) {
+        try {
+            ClassLoader loader = AssetResourceFileFactory.class.getClassLoader();
+            return IOUtils.toByteArray(loader.getResourceAsStream(filePath));
+        } catch (NullPointerException | IOException ioe) {
+            logger.info("Could not read contents of file: " + filePath, ioe);
```
Comment (tlisonbee) [inline src/main/java/com/nike/cerberus/AssetResourceFileFactory.java]: Is logger.error better here?  And are you sure we want to swallow exceptions here?

File: src/main/java/com/nike/cerberus/domain/AssetResourceFile.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.nike.cerberus.domain;
```
Comment (tlisonbee) [inline src/main/java/com/nike/cerberus/domain/AssetResourceFile.java]: Needs license header

File: src/main/java/com/nike/cerberus/AssetResourceFileFactory.java (hunk 1 above)
Comment (sdford) [reply src/main/java/com/nike/cerberus/AssetResourceFileFactory.java]: @tlisonbee No, that was a mistake. Thanks

File: src/main/java/com/nike/cerberus/domain/AssetResourceFile.java (hunk 2 above)
Comment (sdford) [reply src/main/java/com/nike/cerberus/domain/AssetResourceFile.java]: Done

**Comment by fieldju** [comment]
The only thing I don't like about this is there is no option to run the dashboard outside of the JVM which disables things like live reload and live updates, if you make one tweak in the dashboard you have to recompile java. We need a better way when we do dashboard development.

**Comment by sdford** [comment]
@fieldju Good point. I updated `server.js` to allow you to either load the dashboard from the JAR or using Webpack using the `loadDashboardFromCms` variable near the top.

**Pull request merged**